gitlab/table_project_job.go

```diff
@@ -3,12 +3,20 @@ package gitlab
 import (
 	"context"
 	"fmt"
+	"io"
+	"math"
+
 	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
 	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
 	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
 	api "github.com/xanzy/go-gitlab"
 )
 
+type ProjectJob struct {
+	api.Job
+	Trace *string
+}
+
```

Comment:
Is this required, we seemingly only use it to pass `nil` for trace as it's obtained via a hydrate call anyway?

Reply:
Indeed, that comes from my initial tests where I planned to enrich this struct with the data retrieved from the trace API.

Cleanup in progress...

Reply:
Cleaned that up.